docs/pages/zero-trust-access/management/operations/scaling.mdx

```diff
@@ -21,6 +30,53 @@ Set up Teleport with a [High Availability configuration](../../../zero-trust-acc
 | Teleport SSH Nodes connected to Auth Service                          | 50,000                | 2x  4 vCPUs, 16GB RAM | 2x 8 vCPUs, 16GB RAM  | m4.2xlarge         |
 | Teleport SSH Nodes connected to Proxy Service through reverse tunnels | 10,000                | 2x 4 vCPUs, 8GB RAM   | 2x 8 vCPUs, 16+GB RAM | m4.2xlarge         |
 
+### Agent resource requirements
+
+Resource requirements for Teleport Agents vary significantly based on workload, enabled features, and recording configuration.
+The following provides baseline guidance for production deployments.
+
+### Baseline specifications
+
+For production agents, we recommend:
+
+- **Instance type**: AWS t3.small or equivalent (2 vCPUs, 2 GB RAM)
```

Comment:
Where did we get this instance type from? Have we done any testing to confirm that it indeed is satisfactory for normal use of all agent types?